README.md

```diff
@@ -2,11 +2,7 @@
 
 # Project Name
```

Comment:
- [ ] I think it would be better if you add a name and screenshot of your project, as it helps make your repository appear more professional